### ISSUE TITLE
Create Clinicians

### ISSUE THREAD

**Issue by clemens-msupply** [msupply-foundation/open-msupply#2544]
Simple implementation to add Clinician at an Open mSupply store

# Testing 
See #8219 

### Moneyworks Jobcode 🧰
OMS:CIDV
DFAT Resource

**Comment by nisha-dangol** [comment]
Shouldn't we allow adding of the prescriber/clinician in normal dispensary as well? Any reason why should it be limited to `programs` only ? @mark-prins @clemens-msupply

**Comment by clemens-msupply** [comment]
Yes agree should be in the general dispensary UI

**Comment by mark-prins** [comment]
the simple version for this issue please
* add a link to the end of the clinician drop down 'Create new'
* clicking that will pop up a small modal where you can enter code, first name, last name, male/female (is this needed?)
* buttons are `Create` and `Cancel`

**Comment by DhanyaHerath** [comment]
@mark-prins Checking - wouldn't the base solution be allowing for adding/editing clinicians outside of a prescription? In which case we would want: a list of clinicians like we have for patients, and then add/edit functionality there

**Comment by mark-prins** [comment]
the most basic solution would be a way to quickly add a clinician (pop up a modal asking for details) and then see the newly created clinician in the autocomplete / select list where you choose a clinician.

a slightly more ideal solution is to create a list view as well - which would implement a create / edit / delete for all clinicians.
We'd need to keep the create/cancel buttons to redirect as appropriate

I had the most basic solution listed here, as we were trying to get the functionality implemented quickly. if we have enough time, then the list view with edit / delete please

**Comment by andreievg** [comment]
@DhanyaHerath 

> wouldn't the base solution be allowing for adding/editing clinicians outside of a prescription

I agree with @mark-prins about quick way to add clinician with a pop up modal from the same place where clinician is selected, I think it's MVP. 

Then next phase is a list view (which will use pop up modal anyways), and CRUD.

I also think that they will ask for adding clinicians from within prescription (similar to patient).

Btw this will mean modal inside a modal (in patient creation) but we are already doing this (also should return to patient creation modal when clinician is created, a plus to pre select this new clinician)

**Comment by jmbrunskill** [comment]
@roxy-dao Sounds like we might want to support just two gender options for clinicians as well?

**Comment by roxy-dao** [comment]
> [@roxy-dao](https://github.com/roxy-dao) Sounds like we might want to support just two gender options for clinicians as well?

Yes, but will need to figure out a way to configure that... We want to still follow GEDSI standards for the base app after all~
